django_cyverse_auth/authBackends.py

```diff
@@ -473,3 +475,13 @@ def _update_token(self, auth_url, username, token, expiry_time, request=None):
     def _grant_access(self, auth_url, username):
         user_profile = self._user_profile_for_auth(auth_url, username)
         return get_or_create_user(username, user_profile)
+
+    def _lookup_email(self, session):
+        try:
+            user_id = str(session.get_user_id())
+            ks_client = client.Client(session=session)
+            ks_user = ks_client.users.get(user_id)
+            email = str(ks_user.email)
+        except:
+            raise Exception("Cannot get user's email")
```

Comment:
Sure, I think keystone has: https://github.com/openstack/python-keystoneclient/blob/master/keystoneclient/exceptions.py 

Still thinking about how to get the error info.